client/app/components/bibliothek/style.scss

```diff
@@ -0,0 +1,5 @@
+.contentText {
```

Comment:
all class names and ids should use dashes instead of camelCase -> content-text
also just fyi use double-quote in css/scss/html/jsx-templates and singlequote in js code